schema.sql

```diff
@@ -0,0 +1,38 @@
+DROP SCHEMA airbnb CASCADE;
+CREATE SCHEMA airbnb;
+
+-- 10M listings
+CREATE TABLE airbnb.listingDetails (
+  listingId SERIAL PRIMARY KEY,
+  listingName VARCHAR(100) NOT NULL,
```

Comment:
I think you can use less than 100 to save more data space, same fore other things like userName. Thing that won't be super long.